### PR TITLE
WINDUPRULE-19: JBoss WS Package Relocations

### PR DESCRIPTION
**Pull request by jsight**
https://issues.jboss.org/browse/WINDUPRULE-19

### CODE REVIEW

File: rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml [hunk 1]
```diff
@@ -153,5 +153,201 @@
...
+                    <tag>jbossws</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="jboss-eap5-7-java-08100">
+            <when>
+                <javaclass references="org.jboss.wsf.spi.binding.{*}"/>
+            </when>
+            <perform>
+                <hint title="org.jboss.wsf.spi.annotation has been moved to org.jboss.ws.api.binding" effort="1" category-id="mandatory">
```
Comment (mrizzi) [inline rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml]: Replace `org.jboss.wsf.spi.annotation` with `org.jboss.wsf.spi.binding`

File: rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml [hunk 2]
```diff
@@ -153,5 +153,201 @@
...
+                    <tag>jbossws</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="jboss-eap5-7-java-08800">
+            <when>
+                <javaclass references="org.jboss.wsf.common.addressing.{*}" />
+            </when>
+            <perform>
+                <hint title="org.jboss.wsf.common.addressing has been moved to org.jboss.ws.api.annotation" effort="1" category-id="mandatory">
```
Comment (mrizzi) [inline rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml]: Replace `org.jboss.ws.api.annotation` with `org.jboss.ws.api.addressing`

File: rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml [hunk 3]
```diff
@@ -153,5 +153,201 @@
...
+                    <message>
+                        This package has been relocated to `org.jboss.ws.api.annotation` as specified in the
```
Comment (mrizzi) [inline rules-reviewed/eap7/eap5/jboss-eap5-7-java.windup.xml]: Replace `org.jboss.ws.api.annotation` with `org.jboss.ws.api.addressing`

**Comment by mrizzi** [comment]
In the related JIRA, there's `targetTech: eap6+`, maybe it's worth adding the new rules to [jboss-eap5-java.windup.xml](https://github.com/windup/windup-rulesets/blob/master/rules-reviewed/eap6/jboss-eap5/jboss-eap5-java.windup.xml) as well?

**Comment by jsight** [comment]
@mrizzi - It makes no real logical sense, but you are probably right. I'll add some eap6 rules.

**Pull request merged**